src/settings/manager.rs

```diff
@@ -110,14 +110,15 @@ fn fallback_settings_with_backup_file(file_name: &str) -> SettingsStruct {
 
 fn load_settings_from_file(file_name: &str) -> SettingsStruct {
     let result = std::fs::read_to_string(file_name);
-
-    debug!("loaded!");
-    if result.is_err() || cli::manager::is_reset() {
```

Comment:
What about the reset ?

Reply:
It has moved to the function that calls it